src/lib/components/ui/input.svelte

```diff
@@ -74,6 +82,16 @@
 					</label>
 				{/if}
 			</div>
+			{#if search}
+				<div class="delete-button">
+					<Button {dimension} {disabled} variant="secondary">
+						<Delete
+							size={dimension === 'large' ? 32 : dimension === 'small' ? 16 : 24}
+							onclick={() => (value = value.slice(0, -1))}
```

Comment:
This does not really work. But anyways, it should actually erase the whole value not just last character
```suggestion
							onclick={() => (value = "")}
```